### PR TITLE
New Map "Lava Puddles" by smugler5

### PR DESCRIPTION
**Pull request by Avyukt1**
As @JostP told, I have created a new Pull Request with the new screenshot.

### CODE REVIEW

**Review by AnyBlue — CHANGES_REQUESTED** [review]
Change treaure on treasure

**Comment by Avyukt1** [comment]
> Change treaure on treasure

??

File: lava_puddles/map.conf [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+rotation = z
+name = Lava Puddles
+author = smugler5
+h = 140
+team.1.color = red
+team.2.color = blue
+team.2.pos = (101,2,-102)
+team.1 = red
+licence = CC BY-SA 4.0
+hint = Find underground hole to the treaure. Get advantage by collecting all the enemy swords!
```
Comment (JostP) [inline lava_puddles/map.conf:14]:
```suggestion
hint = Find a hole to an underground treasure. Get advantage by collecting all the enemy swords!
```

**Comment by JostP** [comment]
Did you remove the pillar at the corner of the map?

**Comment by Avyukt1** [comment]
> Did you remove the pillar at the corner of the map?

yes

**Review by JostP — APPROVED** [review]
Everything has been fixed, looks good to me :+1:

**Comment by JostP** [comment]
I hope you see now, why we need more days to approve a map. Map reviewer check every detail.

**Comment by JostP** [comment]
Wait one more thing..

File: lava_puddles/map.conf [hunk 2]
```diff
@@ -0,0 +1,14 @@
+team.1.pos = (-98,2,102)
```
Comment (JostP) [inline lava_puddles/map.conf:1]:
```suggestion
team.1.pos = (-98,1,102)

```

File: lava_puddles/map.conf [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+r = 115
+initial_stuff = default:pick_steel,default:dirt 40
+team.2 = blue
+rotation = z
+name = Lava Puddles
+author = smugler5
+h = 140
+team.1.color = red
+team.2.color = blue
+team.2.pos = (101,2,-102)
```
Comment (JostP) [inline lava_puddles/map.conf:11]:
```suggestion
team.2.pos = (101,1,-102)

```

File: lava_puddles/map.conf [hunk 4]
```diff
@@ -0,0 +1,16 @@
+team.1.pos = (-98,1,102)
+
```
Comment (JostP) [inline lava_puddles/map.conf:2]:
```suggestion
team.1.pos = (-98,1,102)
```

File: lava_puddles/map.conf [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+initial_stuff = default:pick_steel,default:dirt 40
+team.2 = blue
+rotation = z
+name = Lava Puddles
+author = smugler5
+h = 140
+team.1.color = red
+team.2.color = blue
+team.2.pos = (101,1,-102)
+
```
Comment (JostP) [inline lava_puddles/map.conf:12]:
```suggestion
team.2.pos = (101,1,-102)
```

**Pull request merged**